Shubham_Sharma/JWT Assignment/UserAuthorization/UserAuthorization/Controllers/AdminController.cs

```diff
@@ -35,15 +35,19 @@ public IActionResult getAll([FromBody] CustomModels.CustomEmployee customEmploye
         [HttpDelete]
         public IActionResult deleteOne([FromBody] CustomModels.CustomEmployee customEmployee)
         {
-
-            if((_idbo.deleteEmp(customEmployee.Username, customEmployee.JwT)))
+            int temp = _idbo.deleteEmp(customEmployee.Username, customEmployee.JwT);
```

Comment:
Still Not Correct We should just receive an Id in case of Delete Not the Whole object

Reply:
I am sending only the token and username( which is a primary key in my case ), and all the other values will be null. The CustomModels' object gets only Username and token, and every other field will be null. From the frontend, I am passing only the username and token.

Reply:
I have made another CustomModel which will only expect Username and token.